docker-entrypoint.sh

```diff
@@ -8,7 +8,7 @@ if [[ "$WAIT_FOR_OPEN_CITY_PROFILE_GRAPHQL_API" = "true" ]]; then
 fi
 
 if [[ "$WAIT_FOR_PARKING_PERMITS_GRAPHQL_API" = "true" ]]; then
-    ADDRESS=$(echo $PARKING_PERMITS_GRAPHQL_API | cut -c 8-)
+    ADDRESS=$(echo $PARKING_PERMITS_GRAPHQL_API | cut -c 8-31)
```

Comment:
@sam-hosseini What is the meaning for this added `31` here?

Reply:
Updated based on our discussions in c18722e 👍🏼

Reply:
👍